cmd/synthetic-monitoring-agent/main.go

```diff
@@ -103,6 +106,8 @@ func run(args []string, stdout io.Writer) error {
 	flags.IntVar(&config.TelemetryTimeSpan, "telemetry-time-span", config.TelemetryTimeSpan, "time span between telemetry push executions per tenant")
 	flags.BoolVar(&config.AutoMemLimit, "enable-auto-memlimit", config.AutoMemLimit, "automatically set GOMEMLIMIT")
 	flags.BoolVar(&config.DisableK6, "disable-k6", config.DisableK6, "disables running k6 checks on this probe")
+	flags.BoolVar(&config.DisableUsageReports, "disable-usage-reports", config.DisableUsageReports, "Disable anonymous usage reports")
+	flags.StringVar(&config.UsageStatsEndpoint, "usage-stats-endpoint", usage.DefaultUsageStatsEndpoint, "usage stats endpoint to send reports to")
```

Comment:
I added this because I needed _some_ mechanism to override this for local development. I don't know if there's ever a time where someone out in the wild would override this, so maybe we can expose this as an environment variable that's hidden from the user?

Reply:
If there's no use case for end users, I wonder if it would be simpler to not have an option for this, and just overwrite it completely for local dev purposes, even if that sounds "ugly" also... But no strong opinion about it, whatever you choose LGTM. :+1:

Reply:
Realistically, the only time this needs to be changed is if we're making changes to the usage stats report. If that happens more then once a year, I'd be surprised. So yeah I agree, let's remove it and let the engineer forcefully change it in code. If this comes up _many_ times, we can reevaluate adding it